docs/android/emulator/mac-with-windows-vm.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Connect to an Android emulator on a Mac from inside a Windows VM"
+description: "Learn how to connect to an Android emulator running on a Mac from inside a Windows virtual machine."
+ms.date: 01/11/2024
+---
+
+# Connect to an Android emulator on a Mac from inside a Windows virtual machine
+
+The Android emulator provides versatile networking capabilities that can be used for different purposes, including connecting to an emulator running on a Mac from inside a Windows virtual machine (VM). This is useful when running Windows in Parallels on a Mac. In this scenario it's required to use the emulators on the Mac since they can't run in Parallels.
+
+> [!IMPORTANT]
+> The address 127.0.0.1 on your development machine corresponds to the emulator's loopback interface.
+
+There are two main approaches for connecting to an Android emulator on a Mac from inside a Windows VM:
+
+1. Using `nc` to perform packet forwarding. For more information, see [Use nc to perform packet forwarding](#use-nc-to-perform-packet-forwarding).
+1. Using `ssh` port forwarding. For more information, see [Use ssh port forwarding](#use-ssh-port-forwarding).
+
+In both cases, the Android Debug Bridge (ADB) is used to connect to the emulator. ADB is a command-line tool that's bundled with the Android SDK Platform Tools package, that lets you communicate with a device. The `adb` command facilitates a variety of device actions, including connecting to devices. For more information about `adb`, see [Android Debug Bridge (adb)](https://developer.android.com/tools/adb) on developer.android.com.
+
+> [!NOTE]
+> You must use an Android Emulator that doesn't include the Google Play Store.
```

Comment:
I meant to ask about this as well - @davidbritch do we know where the `You must use an Android Emulator that doesn't include the Google Play Store.` prohibition comes from? I tried it an image that includes the Play Store and it seemed to work OK, in my 30 seconds of testing. Off hand, I'm not sure why it wouldn't work - what's unique about the Play Store images - so I'm curious about that. The Xamarin doc text here is softer, saying `We recommend using an Android Emulator that does not include the Google Play Store.`.

Reply:
Removing.